src/slice_box.rs

```diff
@@ -12,8 +14,8 @@ pub(crate) struct SliceBox<T> {
 
 impl<T> SliceBox<T> {
     pub(crate) unsafe fn new<'a>(box_: Box<[T]>) -> &'a Self {
-        <Self as typeob::PyTypeObject>::init_type();
-        let type_ob = <Self as typeob::PyTypeInfo>::type_object() as *mut _;
+        // <Self as type_object::PyTypeObject>::init_type();
+        let type_ob = <Self as type_object::PyTypeInfo>::type_object() as *mut _;
```

Comment:
This method returns an uninitialized pointer.
So please use `let type_ob = <Self as type_object::PyTypeObject>::init_type().as_ptr();`